doc/cephadm/troubleshooting.rst

```diff
@@ -176,3 +176,39 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+Manually deploying a MGR daemon
+-------------------------------
+
+cephadm requires a MGR daemon in order to manage the cluster. Please
+follow these steps in order to deploy a MGR ``mgr.hostname.smfvfd`` manually.
+
+Disable the cephadm scheduler, in order to prevent cephadm from removing the new 
+MGR::
+
+  ceph config set mgr mgr/cephadm/pause true
+
+Then get or create the auth entry for the new MGR::
+
+  ceph auth get-or-create mgr.hostname.smfvfd mon "profile mgr" osd "allow *" mds "allow *"
+
+Get the ceph.conf::
+
+  ceph config generate-minimal-conf
+
+Get the container image::
+
+  ceph config get "mgr.hostname.smfvfd" container_image
+
+Create a file ``config-json.json``:
+
+.. code-block: json
+
+  {
+    "config": "# minimal ceph.conf for 8255263a-a97e-4934-822c-00bfe029b28f\n[global]\n\tfsid = 8255263a-a97e-4934-822c-00bfe029b28f\n\tmon_host = [v2:192.168.0.1:40483/0,v1:192.168.0.1:40484/0]\n",
+    "keyring": "[mgr.hostname.smfvfd]\n\tkey = V2VyIGRhcyBsaWVzdCBpc3QgZG9vZi4=\n"
+  }
+
+Deploy the daemon::
+
+  cephadm --image <container-image> deploy --fsid <fsid> --name mgr.hostname.smfvfd --config-json config-json.json
```

Comment:
How do you include this new mgr daemon in the cephadm cluster? It is going to be present in the ceph cluster, (ceph -s) ... but it is not included in the list of cephadm daemons

Reply:
no. that's all. the daemon will be part of `ceph ls` as soon as we're scraping the host the next time.